src/unsafe/writing-unsafe-functions.md

```diff
@@ -0,0 +1,38 @@
+# Writing Unsafe Functions
+
+You can mark your own functions as `unsafe` if they require particular conditions to avoid undefined
+behaviour.
+
+```rust,editable
```

Comment:
Can you fit 20 lines of code on the screen _without scrolling_ in a conference room?

Reply:
Yes.

Reply:
Thanks, I've now added #187 to finally have a way of seeing this at a glance — it confirms that you're right!